### PR TITLE
Hyena nerf

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request
hyena

Updates the Hyena. Changes are as follows:

1. Replaced syndicate cleavers with blastwave jackhammers
2. Downgraded scarlet hardsuits on miners to armored syndicate space suits
3. Added access restrictions to engineering and foreman's office
4. Removed spare headsets from captain and foreman lockers for consistency with other ships
5. Removed codespeak book
6. Pre-built R&D machines
7. Provided miners and engineer with jetpacks
8. Added functional turret control panel

## Why It's Good For The Game

The Hyena is yet another ship in need of touching up as mapping standards shift, and its perpetual power scaling problems continue to present themselves. The cleavers are a common complaint, so hopefully making its mining gear somewhat more in line with other mining ships will help. The Hyena still retains its other gimmicks, such as the bag of explosives.

## Changelog

:cl:
tweak: Updated the Hyena with turret controls and visual updates
balance: Rebalanced the Hyena's mining gear
/:cl:

### CODE REVIEW

**Review by Erikafox — APPROVED** [review]
I would be fine if you went with the idea of jackhammers. It helps it be a mining ship More.

**Comment by Apogee-dev** [comment]
> I would be fine if you went with the idea of jackhammers. It helps it be a mining ship More.

I am of two minds on the jackhammers, honestly. They suit the flavor, and can break down walls at that, but as melee weapons they _do_ have knockback.

I probably will go for it for the flavor, actually.

File: _maps/shuttles/shiptest/syndicate_gorlex_hyena.dmm [hunk 1]
```diff
@@ -363,28 +371,27 @@
 	anchored = 1;
 	icon_state = "hos";
 	name = "captain's locker";
-	req_access_txt = "20"
+	req_access = list(ACCESS_CAPTAIN)
```
Comment (Erikafox) [inline _maps/shuttles/shiptest/syndicate_gorlex_hyena.dmm]: numbers do work they just can't be held in a "". It'd have to be list(20)

**Comment by Erikafox** [comment]
gonna merge as soon as the conflict is resolved

**Comment by Admiral-Gigaton** [comment]
The hyena is dead, i can't... uhh... why did we remove the cleavers again?

**Pull request merged**